### PR TITLE
Adapt with new asset api

### PR DESCRIPTION
**Pull request by XiaoYhun**

### CODE REVIEW

File: src/pages/TrueSightV2/components/TokenListVariants.tsx [hunk 1]
```diff
@@ -1,4 +1,20 @@
+import { useTheme } from 'styled-components'
+
 import Icon from 'components/Icons/Icon'
+import Row from 'components/Row'
+import { ICON_ID } from 'constants/index'
+
+const VARIANTS: { [key: string]: { icon_id: ICON_ID; title: string } } = {
```
Comment (nguyenhoaidanh) [inline src/pages/TrueSightV2/components/TokenListVariants.tsx:7]: duplicate  VARIANTS : src/pages/TrueSightV2/components/SwitchVariantDropdown.tsx

File: src/pages/TrueSightV2/pages/SingleToken.tsx [hunk 2]
```diff
@@ -201,19 +208,30 @@ const TokenDescription = ({ description }: { description: string }) => {
...
-              ? `<span style="color:${
-                  theme.primary
-                }; cursor:pointer; margin-left:4px;" id="hide-token-description-span">${t`Hide`}</span>`
-              : ''),
-        }}
-      />
+      <StyledTokenDescription ref={ref} show={show}>
+        <span
+          dangerouslySetInnerHTML={{
+            __html: linkify(description),
```
Comment (nguyenhoaidanh) [inline src/pages/TrueSightV2/pages/SingleToken.tsx:214]: escapeScriptHtml(linkify(description)) 
or add escapeScriptHtml inside the function linkify

File: src/pages/TrueSightV2/hooks/useKyberAIData.tsx [hunk 3]
```diff
@@ -57,7 +57,7 @@ const kyberAIApi = createApi({
       },
       providesTags: (result, error, arg) => (arg.watchlist === true ? ['myWatchList', 'tokenList'] : ['tokenList']),
     }),
-    //2.
+    //2. TODO
```
Comment (nguyenhoaidanh) [inline src/pages/TrueSightV2/hooks/useKyberAIData.tsx:60]: todo

File: src/pages/TrueSightV2/components/SearchWithDropDown.tsx [hunk 4]
```diff
@@ -322,7 +320,7 @@ const SearchWithDropdown = () => {
   )
   const [history, setHistory] = useLocalStorage<Array<ITokenSearchResult>>('kyberai-search-history')
   const saveToHistory = (token: ITokenSearchResult) => {
-    if (!(history && history.findIndex(t => t.address === token.address && t.chain === token.chain) >= 0)) {
+    if (!(history && history.findIndex(t => t.assetId === token.assetId) >= 0)) {
```
Comment (nguyenhoaidanh) [inline src/pages/TrueSightV2/components/SearchWithDropDown.tsx:323]: history.some(t => t.assetId === token.assetId)

File: src/pages/TrueSightV2/components/TokenListVariants.tsx (hunk 1 above)
Comment (XiaoYhun) [reply src/pages/TrueSightV2/components/TokenListVariants.tsx:7]: they maybe to be modified in future if design changes, should keep them separated for now.

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**